### PR TITLE
allow full-line comments, test with beancount from Mercurial

### PR DESCRIPTION
**Pull request by tbm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ formatting, etc.) can be preserved.
 The syntax of beancount is expected to become slightly less restrictive
 as some missing features are implemented (such as UTF-8 for account names
 and tags on a posting-level).  ledger2beancount aims to be compatible with
-the latest official release of beancount.
+the latest official release of beancount but some functionality may require
```
Comment (zacchiro) [inline README.md]: very minor: a comma is missing before "but"

File: bin/ledger2beancount [hunk 2]
```diff
@@ -354,30 +354,6 @@ sub push_metadata($$$) {
     push_line $depth, pp_metadata($key, $value);
 }
 
-# add a comment line. Wrapper around push_line() for comments
-sub push_comment($$) {
```
Comment (zacchiro) [inline bin/ledger2beancount]: Do you really want to remove `push_comment`? it makes sense as a generic helper. I'd rather keep it, but simplify its logic to just be a `push_line` which additionally adds a heading `;`.

File: docs/manual.md [hunk 3]
```diff
@@ -112,6 +112,25 @@ conversion or if you get an error message from ledger2beancount, please
...
 
+Beancount compatibility
+-----------------------
+
+The syntax of beancount is quite stable but it's expected to become
+slightly less restrictive as some missing features are implemented (such
+as UTF-8 for account names and tags on a posting-level).
+
+ledger2beancount aims to be compatible with the latest official release
+of beancount but some functionality may require an unreleased version of
```
Comment (zacchiro) [inline docs/manual.md]: ditto: comma missing before "but"

File: bin/ledger2beancount (hunk 2 above)
Comment (tbm) [reply bin/ledger2beancount]: Ok, I'll keep it

File: README.md (hunk 1 above)
Comment (tbm) [reply README.md]: Ok

File: docs/manual.md (hunk 3 above)
Comment (tbm) [reply docs/manual.md]: Ok

**Pull request merged**